lib/src/signed_video_internal.h

```diff
@@ -173,13 +173,15 @@ struct _signed_video_t {
   size_t arbitrary_data_size;  // Size of |arbitrary_data|.
 
   bool has_public_key;  // State to indicate if public key is received/added
-  bool sv_test_on;  // State to indicate if public key is received/added
 
   // Handle for vendor specific data. Only works with one vendor.
   void *vendor_handle;
   // Vendor encoders for signing. Only works with one vendor.
   const sv_tlv_tag_t *vendor_encoders;
   size_t num_vendor_encoders;
+
+  // For tests
+  bool sv_test_on;  // State to indicate if public key is received/added
```

Comment:
The comment is incorrect. Rewrite and explain what this flag can be used for.

Reply:
I think it is better to have one single generic flag like we had from the beginning `sv_test_on` or `test_override`.
This flag can then be used to enable/disable other parts if there are needs for that later. There should not be multiple flags for each test functionality. That can easily get out of hands and introduce bugs. In fact, if this should be done properly, this flag should not be possible to set if we build in release mode.

Then you should explain what type of functionality and/or sanity checks that is affected if the flag is turned on.